### PR TITLE
Sparkline: Fixing inability to change lossColor property after runtime change (T1218338)

### PR DESCRIPTION
**Pull request by jdvictoria**

### CODE REVIEW

**Comment by jdvictoria** [comment]
[dxSparkline - Impossible to change lossColor option[T1218338]](https://trello.com/c/MHGOomIL/8532-dxsparkline-impossible-to-change-losscolor-optiont1218338)

File: packages/devextreme/testing/tests/DevExpress.viz.sparklines/sparkline.tests.js [hunk 1]
```diff
@@ -1433,6 +1433,26 @@ QUnit.begin(function() {
         ]);
     });
 
+    QUnit.test('Change winloss lossColor property', function(assert) {
+        const sparkline = this.createSparkline({
+            type: 'winloss',
+            winColor: 'yellow',
+            lossColor: 'blue',
+            firstLastColor: 'pink',
+            winlossThreshold: 4,
```
Comment (AlexanderMoiseev) [inline packages/devextreme/testing/tests/DevExpress.viz.sparklines/sparkline.tests.js:1440]: I believe tests should be as concise as possible. We are testing the runtime change of the lossColor property. Do we need to initialize other properties, such as winLossThreshold or winColor?

File: packages/devextreme/testing/tests/DevExpress.viz.sparklines/sparkline.tests.js [hunk 2]
```diff
@@ -1433,6 +1433,26 @@ QUnit.begin(function() {
         ]);
     });
 
+    QUnit.test('Change winloss lossColor property', function(assert) {
```
Comment (AlexanderMoiseev) [inline packages/devextreme/testing/tests/DevExpress.viz.sparklines/sparkline.tests.js]: please make a test name according to our convention, such as 'lossColor property should be able to change at runtime.'

**Comment by AlexanderMoiseev** [comment]
The main issue is that changes are not reflected after a runtime change. Please adjust the title accordingly

File: packages/devextreme/testing/tests/DevExpress.viz.sparklines/sparkline.tests.js [hunk 3]
```diff
@@ -1433,6 +1433,17 @@ QUnit.begin(function() {
...
 
+    QUnit.test('winloss sparkline lossColor should change at runtime', function(assert) {
+        const sparkline = this.createSparkline({
+            type: 'winloss',
+            lossColor: 'blue',
+        });
+
+        sparkline.option({ lossColor: 'green' });
+
+        assert.strictEqual(sparkline.option('lossColor'), 'green', 'lossColor should be green after runtime change');
```
Comment (AlexanderMoiseev) [inline packages/devextreme/testing/tests/DevExpress.viz.sparklines/sparkline.tests.js]: this condition won't fail if you remove your fix, i suppose. probably we have to use 'checkCustomizePoint' here, please check. Btw, following TDD principles would be helpful in this scenario

**Review by AlexanderMoiseev — CHANGES_REQUESTED** [review]

File: packages/devextreme/testing/tests/DevExpress.viz.sparklines/sparkline.tests.js [hunk 4]
```diff
@@ -1433,6 +1433,23 @@ QUnit.begin(function() {
         ]);
     });
 
+    QUnit.test('winloss sparkline lossColor should change', function(assert) {
```
Comment (AlexanderMoiseev) [inline packages/devextreme/testing/tests/DevExpress.viz.sparklines/sparkline.tests.js]: i left a comment before - 'The main issue is that changes are not reflected after a runtime change. Please adjust the title accordingly' - the same goes for the test name, as i already suggested here https://github.com/DevExpress/DevExtreme/pull/26837#discussion_r1513859960

File: packages/devextreme/testing/tests/DevExpress.viz.sparklines/sparkline.tests.js [hunk 5]
```diff
@@ -1433,6 +1433,23 @@ QUnit.begin(function() {
         ]);
     });
 
+    QUnit.test('winloss sparkline lossColor should be updated after runtime change', function(assert) {
+        const sparkline = this.createSparkline({
+            type: 'winloss',
+            lossColor: 'blue',
+            winlossThreshold: 4,
```
Comment (AlexanderMoiseev) [inline packages/devextreme/testing/tests/DevExpress.viz.sparklines/sparkline.tests.js:1440]: just wondering - is settng 'winlossThreshold' necessary for this tests?

Comment (jdvictoria) [reply packages/devextreme/testing/tests/DevExpress.viz.sparklines/sparkline.tests.js:1440]: image
Without 'winlossThreshold' the test doesn't run as intended. Still trying to figure out how to omit it.

File: packages/devextreme/testing/tests/DevExpress.viz.sparklines/sparkline.tests.js [hunk 6]
```diff
@@ -1433,6 +1433,23 @@ QUnit.begin(function() {
         ]);
     });
 
+    QUnit.test('winloss sparkline lossColor should be updated after runtime change', function(assert) {
```
Comment (AlexanderMoiseev) [inline packages/devextreme/testing/tests/DevExpress.viz.sparklines/sparkline.tests.js]: please add ticket number to the test's name

**Review by AlexanderMoiseev — DISMISSED** [review]

**Review by alexanderPolosatov — APPROVED** [review]

**Review by AlexanderMoiseev — APPROVED** [review]

**Pull request merged**